src/main/java/com/blog/web/mappers/ArticleMapper.java

```diff
@@ -0,0 +1,31 @@
+package com.blog.web.mappers;
+
+import com.blog.web.dto.ArticleDto;
+import com.blog.web.models.Article;
+
+public class ArticleMapper {
+    public static Article mapToArticle(ArticleDto articleDto) {
+        Article article = Article.builder()
```

Comment:
ur using the builder but looks like all you really fill is ArticleDto consider wrapping it/having articleDto be a parameter in constructor, or a factory that generates articles based on article dto